src/app/Caches/Create/ConfigurationBasic.tsx

```diff
@@ -0,0 +1,231 @@
+                    isInline
+                    isRequired
+                    fieldId="topology"
+                >
+                    <Radio
+                        name="radio"
+                        id="distributed"
+                        onChange={() => setTopology('distributed')}
+                        isChecked={topology === 'distributed'}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>Distributed</Text>
+                            </TextContent>
+                        }
+                    />
+                    <Radio
+                        name="radio"
+                        id="replicated"
+                        onChange={() => setTopology('replicated')}
+                        isChecked={topology === 'replicated'}
+                        label={
+                            <TextContent>
+                                <Text component={TextVariants.h3}>Replicated</Text>
+                            </TextContent>
+                        }
+                    />
+                </FormGroup>
+            </React.Fragment>
+        );
+    };
+
+    // Options for Number Owners
+    const optionsNumberOwners = [
+        <SelectOption key={0} value="Select" isPlaceholder />,
+        <SelectOption key={1} value="1" />,
+        <SelectOption key={2} value="2" />,
+        <SelectOption key={3} value="3" />,
+        <SelectOption key={4} value="4" />,
+        <SelectOption key={5} value="5" />,
+        <SelectOption key={6} value="6" />,
+        <SelectOption key={7} value="7" />,
+        <SelectOption key={8} value="8" />,
+        <SelectOption key={9} value="9" />,
+        <SelectOption key={10} value="10" />,
+    ];
+
+    // Form Number of Owners
+    const formNumberOwners = () => {
+        return (
+            <React.Fragment>
+                <FormGroup
+                    label="Number of owners"
+                    fieldId='field-number-owners'
+                    isRequired={topology === 'distributed'}> {/*Required when topology is distributed */}
+                    <Tooltip content="Tooltip for number of owners"><OutlinedQuestionCircleIcon /></Tooltip>
+                    <Select
+                        variant={SelectVariant.single}
+                        aria-label="Select Input"
+                        onToggle={onToggleNumberOwners}
+                        onSelect={onSelectNumberOwners}
+                        selections={selectedNumberOwners}
+                        isOpen={isOpenNumberOwners}
+                        aria-labelledby="toggle-id-number-owners"
+                    >
+                        {optionsNumberOwners}
+                    </Select>
+                </FormGroup>
+            </React.Fragment>
+        )
+    }
+
+    // Options for Encoding Cache
+    const optionsEncodingCache = [
+        <SelectOption key={0} value="JSON" isPlaceholder />,
```

Comment:
the encoding options are not ok.
the encoding type is in InfinispanRefData.EncodingType. Same for the type of caches